src/libs/ajax/loaded-data/useLoadedData.test.ts

```diff
@@ -35,7 +36,7 @@ describe('useLoadedData hook', () => {
       });
     });
     const hookResult2: UseLoadedDataResult<TestData> = hookRender.result.current;
-    await hookRender.waitForNextUpdate();
+    await act(() => updatePromise);
```

Comment:
Will this resolve `updatePromise`?

Reply:
It will wait for updatePromise to resolve.